coderkevin/fresh-data-wp-plugin-example
# resourceName: Adapt example code

This adapts the example code for resourceName and operations.

To Test:

0. Ensure you've built fresh data and done `npm link` there.
1. `npm install`, `npm link fresh-data`, `npm start`
2. Visit Fresh Data menu item and verify it pulls a page of tags and displays them. (just like before).
